.phan/config.php

```diff
@@ -307,6 +311,9 @@
     //       to `exclude_analysis_directory_list`.
     'exclude_analysis_directory_list' => [
         'vendor/',
+        'config/',
+        'src/Factory',
```

Comment:
Why excluding source?

Reply:
Phan does not understand the factory call, and the configurations with the di-container, throwing false errors.